common/src/main/java/io/netty/util/internal/StringUtil.java

```diff
@@ -30,6 +33,11 @@
     private static final String[] BYTE2HEX_PAD = new String[256];
     private static final String[] BYTE2HEX_NOPAD = new String[256];
     private static final String EMPTY_STRING = "";
+    public static final char DOUBLE_QUOTE = '\"';
+    public static final char COMMA = ',';
+    public static final char LINE_FEED = '\n';
+    public static final char CARRIAGE_RETURN = '\r';
+    public static final String DOUBLE_QUOTE_STRING = String.valueOf(DOUBLE_QUOTE);
```

Comment:
I think this is only needed in the test case `DefaultTextHeadersTest`.  Can we move it there?